src/components/Card/Card.js

```diff
@@ -38,7 +38,10 @@ const Card = ({
 
   const content = (
     <>
-      <div className="wfp--photo-cardnew__background" style={style} />
+      {
+        kind === 'overlay' ? <div className="wfp--photo-cardnew__background" style={style} /> : null
```

Comment:
For the classNames you can use the `${prefix}--class pattern`.

Reply:
okay